### PR TITLE
styleKit  vars: border_radius (#660)

### PR DESCRIPTION
**Pull request by FredLL-Avaiga**
border_radius: int in theme and str with px in CSS
input button height: str
fix " issue in HTML :root
Type the stylekit variables dict

### CODE REVIEW

File: gui/src/utils/index.ts [hunk 1]
```diff
@@ -47,11 +47,11 @@ interface StyleKitVars {
     // Set main font family
     fontFamily: string;
     // DEFINING SHAPES
-    // Base border radius
+    // Base border radius in px
```
Comment (FabienLelaquais) [inline gui/src/utils/index.ts:50]: I love that you mention the unit we're dealing with.
I wish that would appear everywhere (like in the inputButtonHeight property right after)

**Review by FabienLelaquais — APPROVED** [review]
Looks good!

File: gui/src/utils/index.ts (hunk 1 above)
Comment (FredLL-Avaiga) [reply gui/src/utils/index.ts:50]: inputButtonHeight is a string and the unit can be anything

Comment (FabienLelaquais) [reply gui/src/utils/index.ts:50]: That's my point. CSS unit I guess.

Comment (FredLL-Avaiga) [reply gui/src/utils/index.ts:50]: added in css size unit

Comment (FredLL-Avaiga) [reply gui/src/utils/index.ts:50]: Not sure about the wording, if you have a better proposal ?

Comment (FabienLelaquais) [reply gui/src/utils/index.ts:50]: In my opinion the wording should be the one we've used in the elements' doc.
"The &lt;whatever&gt;, in CSS units, of this &lt;bla&gt;;."

File: src/taipy/gui/_default_config.py [hunk 2]
```diff
@@ -62,11 +62,11 @@
         "font_family": "Lato, Arial, sans-serif",
 
         # DEFINING SHAPES
-        # Base border radius
-        "border_radius": "8px",
+        # Base border radius in px
+        "border_radius": 8,
```
Comment (dinhlongviolin1) [inline src/taipy/gui/_default_config.py:66]: Thanks for this addition. TS has been screaming at me on the studio side on this value

**Review by dinhlongviolin1 — APPROVED** [review]

**Pull request merged**